src/OrbitGl/LiveFunctionsDataView.cpp

```diff
@@ -128,9 +128,9 @@ void LiveFunctionsDataView::DoSort() {
     return;
   }
   bool ascending = sorting_orders_[sorting_column_] == SortingOrder::kAscending;
-  std::function<bool(int a, int b)> sorter = nullptr;
+  std::function<bool(uint64_t a, uint64_t b)> sorter = nullptr;
 
-  const std::vector<FunctionInfo>& functions = functions_;
+  const auto& functions = functions_;
```

Comment:
Can you state the type? Even if this is just a trick for the macros above, it's not immediately clear this is a map.

Reply:
done.